src/OrbitGl/GraphTrack.cpp

```diff
@@ -284,10 +290,31 @@ void GraphTrack<Dimension>::DrawSeries(PrimitiveAssembler& primitive_assembler,
     uint64_t current_time = std::max(current_it->first, min_tick);
     auto next_it = std::next(current_it);
     uint64_t next_time = std::min(next_it->first, max_tick);
-    DrawSingleSeriesEntry(primitive_assembler, current_time, next_time,
-                          normalized_cumulative_values, z);
+
+    if (current_it == entries.begin()) {
+      aggr.StartNewEntry(current_time, next_time, normalized_cumulative_values);
+    } else {
+      // If the size of the current entry is less than a pixel
```

Comment:
Expand this with something like:

`... "and fits in the same pixel than the previous entry."`